docs/md/modules-covid.md

```diff
@@ -0,0 +1,166 @@
+# Covid namespace
+
+WorkFlow Launcher (WFL) runs within the `covid` namespace to process workflows using the `Sarscov2IlluminaFull` pipeline. 
+
+Workflow Launcher uses a model which includes a source, a sink and an executor. Within the `covid` namespace, the source at the time of this writing is either directly from the Terra Data Repository (`TerraDataRepo`), or a list of snapshots which are passed to WFL (`TDR Snapshots`). 
```

Comment:
Is it worth calling out here that a COVID workload will continue to poll the source for inputs until it is stopped?

Not the same thing but AoU docs have a call-out re: the continuous nature of the pipeline.
https://broadinstitute.github.io/wfl/modules-arrays.html

Reply:
I added it to #2 of the life-cycle of a workload.